package.json

```diff
@@ -11,8 +11,9 @@
     "nodes": {
       "Entity": "src/nodes/NGSI/entity/entity.js",
       "Dataset": "src/nodes/NGSI/dataset/dataset.js",
-      "Update":  "src/nodes/NGSI/update/update.js",
-      "Subscription": "src/nodes/NGSI/subscription/subscription.js",
+      "Update": "src/nodes/NGSI/update/update.js",
+      "Subscriber": "src/nodes/NGSI/subscriber/subscriber.js",
```

Comment:
please check the current package.json ...

Reply:
Right, my bad, was a bit too fast on the change there.

Reply:
the node definition is duplicated ...

Reply:
Holy... Good thing I'm going on holidays tomorrow, guess my head is somewhere else today. Sorry, I'll fix it.